include/aws/http/request_response.h

```diff
@@ -694,8 +694,10 @@ struct aws_http_message *aws_http2_message_new_response(struct aws_allocator *al
  * Create an HTTP/2 message from HTTP/1.1 message.
  * pseudo headers will be created from the context and added to the headers of new message.
  * Normal headers will be copied to the headers of new message.
- * Note: if `host` exist, it will stay and `:authority` will be added using the information.
- * `:scheme` is default to be "https". If a different scheme wants to be used, create the HTTP/2 message directly
+ * Note:
+ *  - if `host` exist, it will be removed and `:authority` will be added using the information.
+ *  - `:scheme` is always default to be "https". If a different scheme wants to be used,
```

Comment:
nit: "`:scheme` always defaults to "https". To use a different scheme create the HTTP/2 message directly"